### ISSUE TITLE
ErrorCodeException

### ISSUE THREAD

**Issue by lines27** [planbnet/livisi_unofficial#78]
## System Information

version | core-2024.1.5
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.6
os_name | Linux
os_version | 6.1.71-haos
arch | x86_64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4992
Installed Version | 1.34.0
Stage | running
Available Repositories | 1392
Downloaded Repositories | 5

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Easee EV Charger</summary>

component_version | 0.9.55
-- | --
reach_easee_cloud | ok
connected2stream | true

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.4
-- | --
update_channel | stable
supervisor_version | supervisor-2024.01.1
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 234.0 GB
disk_used | 8.3 GB
healthy | true
supported | true
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | Terminal & SSH (9.8.1), Studio Code Server (5.15.0), Node-RED (17.0.3), evcc (0.123.9)

</details>

<details><summary>Dashboards</summary>

dashboards | 3
-- | --
resources | 3
views | 6
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 9. Februar 2024 um 12:29
-- | --
current_recorder_run | 9. Februar 2024 um 13:29
estimated_db_size | 150.92 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Hello planbnet,
due to I don't know if this error-message is related to the issue "[Get AttributeError after installation]"(https://github.com/planbnet/livisi_unofficial/issues/61#top)
I open a new issue.
I installed the newest version 1.7.4 on 2024-02-09 and got the following error on 2024-02-11. 
Up to now only once.

  File "/config/custom_components/livisi/livisi_connector.py", line 155, in _async_request
    raise ErrorCodeException(response["errorcode"])
custom_components.livisi.livisi_errors.ErrorCodeException

Details see below.

### Reproduction steps

1. Install newest version 1.7.4
2. Restart system
3.
...

### Debug logs

```text
Logger: custom_components.livisi
Source: helpers/update_coordinator.py:300
Integration: Livisi Unofficial (documentation, issues)
First occurred: 11. Februar 2024 um 13:30:04 (1 occurrences)
Last logged: 11. Februar 2024 um 13:30:04

Unexpected error fetching Livisi devices data:
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 300, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/livisi/coordinator.py", line 83, in _async_update_data
    return await self.async_get_devices()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/livisi/coordinator.py", line 103, in async_get_devices
    devices = await self.aiolivisi.async_get_devices()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/livisi/livisi_connector.py", line 194, in async_get_devices
    devices, capabilities, messages, rooms = await asyncio.gather(
                                             ^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/livisi/livisi_connector.py", line 101, in async_send_authorized_request
    return await self._async_request(method, url, payload, auth_headers)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/livisi/livisi_connector.py", line 155, in _async_request
    raise ErrorCodeException(response["errorcode"])
custom_components.livisi.livisi_errors.ErrorCodeException
```

### Diagnostics dump

Diagnostic function for this integration not available

**Comment by planbnet** [comment]
Yes, this is the same error as in #61 (as the release 1.7.4 says - this was just improved logging)
The error sahould not be fixed in 1.7.5
